Update Mock up mangga di review

fazzabd opened the pull request.

fazzabd committed:
Add files via upload

MaulanaIbrahim commented:
sedikit merivew untuk yang mockup dari sisi user

![image](https://user-images.githubusercontent.com/28476564/31366424-fd327bd2-ad9a-11e7-8ab7-2c6c727d41cc.png)

ada penamaan yang masih ambigu yaitu event location . karena konteks yang di ambil untuk mengisi form tersebut adalah yang mewadahi event tersebut sedangkan di dalam kbbi kata lokasi sendiri tidak merepresentasikan itu. saran kenapa tidak di ganti event place?

fazzabd committed:
update mock up

ada beberapa yg belum beres di mock up admin, belum bisa tambah lokasi

fazzabd committed:
Add files via upload

membenarkan mock up hasil review, yaitu pemilihan waktu

fazzabd committed:
Add files via upload

update mock up, benerin dari hasil review, ganti cara pilih jam, dan rapihin yg lain juga

fazzabd committed:
Merge pull request #7 from A2JTK16/fazzabd-patch-1-mock-up

update mock up
